cmd/podman/rmi.go

```diff
@@ -63,13 +63,14 @@ func rmiCmd(c *cli.Context) error {
 	for _, arg := range imagesToDelete {
 		image, err := runtime.GetImage(arg)
 		if err != nil {
-			return errors.Wrapf(err, "could not get image %q", arg)
+			fmt.Printf("could not get image %q: %v\n", arg, err)
 		}
 		id, err := runtime.RemoveImage(image, c.Bool("force"))
 		if err != nil {
-			return errors.Wrapf(err, "error removing image %q", id)
+			fmt.Printf("%v\n", err)
```

Comment:
No need to wrap error, runtime.RemoveImage already returns good errors. Wrapping it would just be repeating the error.